api/server/structs/conversions.go

```diff
@@ -1088,10 +1156,10 @@ func AttesterSlashingsToConsensus(src []*AttesterSlashing) ([]*eth.AttesterSlash
 		if s == nil {
 			return nil, server.NewDecodeError(errNilValue, fmt.Sprintf("[%d]", i))
 		}
-		if s.Attestation1 == nil {
+		if s.Attestation1 == nil || s.Attestation1.Data == nil {
```

Comment:
Can you split this so that one error returns `"[%d].Attestation1"` and the other one `"[%d].Attestation1.Data`?